src/renderer/pages/Governance/ui/Governance.tsx

```diff
@@ -149,17 +161,31 @@ export const Governance = () => {
           />
         )}
 
+      {showRevoteModal &&
+        nonNullable(network) &&
```

Comment:
What is your opinion maybe add possibility to provide several params for nonNullable?

Reply:
I've tried to write similar helpers, but typescript type narrowing don't work in such cases
Idea was smth like

```ts
type NonNullableTuple<Tuple extends [...any[]]> = {
  [Index in keyof Tuple]: NonNullable<Tuple[Index]>;
} & { length: Tuple['length'] };

function nonNullables<const T extends unknown[]>(values: T): values is NonNullableTuple<T> {
  return values.every((value) => value !== null && value !== undefined);
}
```